lib/parseBinaryGltf.js

```diff
@@ -47,10 +47,11 @@ function parseBinaryGltf(data) {
     var gltf = JSON.parse(scene);
     addPipelineExtras(gltf);
 
-    // Extract binary body
-    var body = data.slice(20 + sceneLength);
+    //Extract binary body
+    byteOffset += sceneLength;

```

Comment:
Why did you make this change? Also byteOffset is not declared so this is probably throwing jsHint errors.